build.py

```diff
@@ -462,6 +462,8 @@
     'TestFoundation/Resources/NSStringTestData.txt',
     'TestFoundation/Resources/NSString-UTF16-BE-data.txt',
     'TestFoundation/Resources/NSString-UTF16-LE-data.txt',
+    'TestFoundation/Resources/NSString-UTF32-BE-data.txt',
```

Comment:
They're actually there. Because these files have BOM in them, Github is detecting them to be binary files. If you go ahead and click on "Files Changed" above, you would see them listed as binary files..